tests/helpers.bash

```diff
@@ -13,6 +13,19 @@ OCI=$(${BUILDAH_BINARY} info --format '{{.host.OCIRuntime}}' || command -v runc
 # Default timeout for a buildah command.
 BUILDAH_TIMEOUT=${BUILDAH_TIMEOUT:-300}
 
+# Safe reliable unchanging test image
+SAFEIMAGE_REGISTRY=${SAFEIMAGE_REGISTRY:-quay.io}
+SAFEIMAGE_USER=${SAFEIMAGE_USER:-libpod}
+SAFEIMAGE_NAME=${SAFEIMAGE_NAME:-testimage}
+SAFEIMAGE_TAG=${SAFEIMAGE_TAG:-20221018}
+SAFEIMAGE="${SAFEIMAGE:-$SAFEIMAGE_REGISTRY/$SAFEIMAGE_USER/$SAFEIMAGE_NAME:$SAFEIMAGE_TAG}"
+
+# Prompt to display when logging buildah commands; distinguish root/rootless
+_LOG_PROMPT='$'
+if [ $(id -u) -eq 0 ]; then
+    _LOG_PROMPT='#'
+fi
+
```

Comment:
This is misleading, because it's not actually used anywhere (it's part of #4320,  but only an incomplete part). I have a strong aversion to unused code, because it can cost time for someone in the future. Would you mind removing it and re-pushing? Or adding the other missing part of #4320?
```diff
-        echo "\$ $BUILDAH_BINARY $*"
+        echo "${_LOG_PROMPT} $BUILDAH_BINARY $*"
```

Reply:
I will remove this chunk